### ISSUE TITLE
Panel Profiles error.

### ISSUE THREAD

**Issue by MartinMilesi** [phoenixbyrd/Termux_XFCE#18]
Hello, the configurator for panel profiles wont start, it needs psutil you can install it with pip.

pip install psutil

with that it works well.

Im using this in a Motorola moto g200 and work well full office in the work :D 

Here some tips for you:

You could add a custom action i  thunar to send files to android wit thunar. You can use this command:

xdg-open %f

You could also make some launcher in the bar or desktop to open android app. Here some examples:

whatsapp
am start --user 0 -n com.whatsapp/.Main

termux-x11-configuration
am start --user 0 -n com.termux.x11/.LoriePreferences

Activity Launcher
am start --user 0 -n de.szalkowski.activitylauncher/.MainActivity

You can use the "activity launcher to know the names of the classes to start apps. i dont remember if you can install that app from the google play store or fdroid.

you could also add a shortuc to the appareance manager of the proot so all apps get the same theme in proot that whe are using in termux.

good luck and good work, please if you have the time make an updater, your script saved me a lot of time an trouble so thanks!

**Comment by phoenixbyrd** [comment]
I'm not sure what the configurator is or means or panel profiles. I'm not that well versed in everything.

I'm not sure what xdg-open %f does, I tried running it in terminal but nothing happened. 

The launcher things, that would be something personal.  I tried running am start --user 0 -n com.termux.x11/.LoriePreferences, but nothing happened except it gave this output

Starting: Intent { cmp=com.termux.x11/.LoriePreferences }

I think it would be best to let the users decide what they want to add to it based on their needs. 

proot doesn't have a desktop installed on it so there's no appearance manager there. I tried figuring it out and doing my best to make it match termux settings through command line.
